### ISSUE TITLE
Widget is not updating after changes in notes

### ISSUE THREAD

**Issue by Wurzelmann** [BijoySingh/Scarlet-Notes#104]
My notes widgets are not updating properly after changing the contents of its notes. They just remain the same and hours later (haven't really measured how long it takes, but multiple hours) they update its contents.

App version. 5.9.7
Android version: 9

**Comment by BijoySingh** [comment]
Yes this was resolved in a later commit. Should be fixed on play store and once FDroid picks up the latest release on FDroid as well.
